### PR TITLE
add command to change registered multiaddrs for miner

### PR DESCRIPTION
**Pull request by whyrusleeping**
This lets you set multiaddrs for your miner, but we don't use them quite yet. that needs some changes in the markets first

### CODE REVIEW

File: cmd/lotus-storage-miner/info.go [hunk 1]
```diff
@@ -248,3 +254,71 @@ func sectorsInfo(ctx context.Context, napi api.StorageMiner) error {
...
+
+		gasLimit := cctx.Int64("gas-limit")
+
+		smsg, err := api.MpoolPushMessage(ctx, &types.Message{
+			To:       maddr,
+			From:     minfo.Worker,
+			Value:    types.NewInt(0),
+			GasPrice: types.NewInt(1),
+			GasLimit: gasLimit,
+			Method:   18,
```
Comment (Kubuxu) [inline cmd/lotus-storage-miner/info.go:316]: We should use enum from specs-actors here.

Comment (whyrusleeping) [reply cmd/lotus-storage-miner/info.go:316]: i would if it were set

File: cmd/lotus-storage-miner/info.go [hunk 2]
```diff
@@ -4,26 +4,32 @@ import (
...
+	"github.com/filecoin-project/lotus/chain/actors"
 	"github.com/filecoin-project/lotus/chain/types"
 	lcli "github.com/filecoin-project/lotus/cli"
 )
 
 var infoCmd = &cli.Command{
 	Name:  "info",
 	Usage: "Print storage miner info",
+	Subcommands: []*cli.Command{
+		infoSetAddrsCmd,
```
Comment (magik6k) [inline cmd/lotus-storage-miner/info.go:31]: I'd put thin under a new `actor` namespace

File: cmd/lotus-storage-miner/actor.go [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+		}
+
+		params, err := actors.SerializeParams(&miner.ChangeMultiaddrsParams{addrs})
+		if err != nil {
+			return err
+		}
+
+		gasLimit := cctx.Int64("gas-limit")
+
+		smsg, err := api.MpoolPushMessage(ctx, &types.Message{
```
Comment (Kubuxu) [inline cmd/lotus-storage-miner/actor.go:77]: This error is not checked.

**Comment by Kubuxu** [comment]
Also lint warnings.

**Pull request merged**